tests/Unit/Domain/BoundaryConditions/Test_Helpers.cpp

```diff
@@ -16,18 +17,28 @@
 namespace {
 namespace helpers = TestHelpers::domain::BoundaryConditions;
 
-SPECTRE_TEST_CASE("Unit.Domain.BoundaryConditions.Periodic", "[Unit][Domain]") {
+SPECTRE_TEST_CASE("Unit.Domain.BoundaryConditions.Helpers", "[Unit][Domain]") {
```

Comment:
[minor, non-binding] I feel it would be slightly more descriptive to rename "Helpers" -> "GenericBcs" or "BcTemplates" or similar? (for file name and for test name)